.golangci.yml

```diff
@@ -275,6 +275,7 @@ linters-settings:
       - name: max-public-structs
         exclude: ["TEST"]
         arguments: [5]
+        disabled: true
```

Comment:
Disabled this linter because it was reporting false positives. E.g: The `internal/utils/status` package has only 2 public structs but the linter still reports it:

<img width="989" alt="Screenshot 2024-11-12 at 9 15 34 PM" src="https://github.com/user-attachments/assets/7ec39f13-8cb7-4308-a5db-458b8e02ead9">

Reply:
You're right, there is a bug in the v1.62.0 linter that provides false-positives. It was already fixed in our workflow, could you please rebase on master? Let's keep this linter enabled.